ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/v2/NewUiKitEditTextMutator.kt

```diff
@@ -0,0 +1,119 @@
+package org.p2p.uikit.components.edittext.v2
+
+import androidx.annotation.DrawableRes
+import androidx.core.view.isVisible
+import androidx.core.widget.addTextChangedListener
+import android.content.res.ColorStateList
+import android.text.Editable
+import android.text.TextWatcher
+import android.text.method.DigitsKeyListener
+import android.widget.EditText
+import org.p2p.core.common.TextContainer
+import org.p2p.core.common.bind
+import org.p2p.core.utils.emptyString
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetUiKitEditTextNewBinding
+import org.p2p.uikit.utils.SimpleTagTextWatcher
+import org.p2p.uikit.utils.getColor
+
+/**
+ * Allows to change state of the view by settings text / listeners and other stuff
+ */
+class NewUiKitEditTextMutator(
```

Comment:
setters
you can use this class if you want to change (mutate) the state of the view.